### ISSUE TITLE
Initial docs

### ISSUE THREAD

**Issue by goodboy** [sangoma/pytestlab#3]
We have a set of internal docs which need to be reworked and the `pytestlab` specific stuff needs to come here.

I was debating between using the new github wiki stuff vs. sphinx + readthedocs.
@vodik any opinion on that?

**Comment by jhermann** [comment]
Use Sphinx for the "proper" docs – with support for code highlighting, cross-references(!), and other stuff you'll miss in the wiki. Then open the wiki for user contributions – for many people, that is a lower entry barrier, and you get the best of both worlds. Refactor wiki pages into the docs where appropriate.

**Comment by vodik** [comment]
Yeah, that's close to what we already have @jhermann, just need to seperate out what we dumped here from other internal/product specific pieces. This and #1 is going to get my full focus very shortly as I polish up stuff on the development branch and detangle the last few pieces that are product specfic here.

**Comment by tacerus** [comment]
Hi there! Any news on the documentation?
